x-pack/plugins/monitoring/public/components/logstash/listing/listing.js

```diff
@@ -114,12 +115,12 @@ class ListingUI extends PureComponent {
     const columns = this.getColumns();
     const flattenedData = data.map(item => ({
       ...item,
-      name: item.logstash.name,
-      cpu_usage: item.process.cpu.percent,
-      load_average: item.os.cpu.load_average['1m'],
-      jvm_heap_used: item.jvm.mem.heap_used_percent,
-      events_ingested: item.events.out,
-      version: item.logstash.version,
+      name: get(item, 'logstash.name'),
+      cpu_usage: get(item, 'process.cpu.percent'),
+      load_average: get(item, 'os.cpu.load_average.1m'),
```

Comment:
I just tested this by removing the `logstash_stats.os.cpu` field in `.monitoring-logstash-*` docs with `type: logstash_stats`. The result is that the load average is shown as `0.00` in the UI. This could be misleading as users could interpret it as "there's no or really low load on the system".

Could we make it so that the default here is set to a sentinel value and then check for that value in https://github.com/elastic/kibana/pull/34939/files#diff-41737f3c90b33f58268178cc5aa38d20R60 and display something like `N/A`?

Reply:
Yes, great thought!